store/README.md

```diff
@@ -127,3 +127,23 @@ type Store struct {
 ```
 
 `Store.Store` is a `dbadapter.Store` with a `dbm.NewMemDB()`. All `KVStore` methods are reused. When `Store.Commit()` is called, new `dbadapter.Store` is assigned, discarding previous reference and making it garbage collected.
+
+## Table
```

Comment:
This seems to be a duplicate of what lives in docs/core/store.md.

Reply:
Yeah indeed as I pointed out in https://github.com/cosmos/cosmos-sdk/pull/9751#discussion_r674730910
In this regard, I'm not sure if that's relevant to keep this file, I would appreciate some thoughts on that.

Reply:
Moving the orm package to x/group for now as pointed out here https://github.com/cosmos/cosmos-sdk/pull/9751#pullrequestreview-713063087 so this is not really relevant anymore